microsoft-edge/devtools-guide-chromium/memory-problems/share-traces.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Share memory traces
+description: Learn to export and import memory traces in DevTools to get help when investigating issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/06/2023
+---
+# Share memory traces
+
+The **Memory** tool records runtime data about your web page. Exploring the recorded data makes it possible to improve your web page’s memory usage. Additionally, the recorded data can be exported to files on disk. The exported files are called traces.
```

Comment:
```suggestion
The **Memory** tool records runtime data about your web page. Exploring the recorded data makes it possible to improve your web page's memory usage. Additionally, the recorded data can be exported to files on disk. The exported files are called _traces_.
```
https://learn.microsoft.com/en-us/contribute/markdown-reference#apostrophes-and-quotation-marks